packages/service-clients/odsp-client/src/interfaces.ts

```diff
@@ -68,31 +67,26 @@ export interface OdspContainerServices {
 /**
  * Since ODSP provides user names and email for all of its members, we extend the
  * {@link @fluidframework/protocol-definitions#IMember} interface to include this service-specific value.
- * @alpha
+ * It will be returned for all audience members connected.
+ *
+ * @internal
  */
-export interface OdspUser extends IUser {
+export interface OdspMember extends IMember {
+	/**
+	 * The object ID or object Identifier (oid). It is a unique identifier assigned to each user, group, or other entity within AAD or another Microsoft 365 service.
+	 * Overrides the documentation from IMember.userId.
```

Comment:
Probably don't need to explicitly mention that this is overriding the documentation.

I think here we expect it to specifically refer to a user, right?

```suggestion
	 * The object ID (oid) for the user, unique among each individual user connecting to the session.
```